src/Button/Button.js

```diff
@@ -76,6 +83,18 @@ const Button = React.forwardRef(({
         }
         return content;
     };
+
+    const renderIcon = () => {
+        const content = glyph ? (
+            <Icon
+                {...iconProps}
+                ariaHidden
+                glyph={glyph} />
+        ) : null;
+
```

Comment:
Does it work to just do something like:
```
const renderIcon = () => {
return ( glyph && <Icon props here />
)
```
}

Reply:
lint complains `Declare only one React component per file`

Not sure if it's cleaner to disable it / if we want to 🤷‍♂️

Reply:
Eh, I think it's fine the way you have it